src/data_provider/src/packages/brewWrapper.h

```diff
@@ -47,6 +47,9 @@ class BrewWrapper final : public IPackageWrapper
                     break;
                 }
             }
+
+            // Some brew packages have the version in the name separated by a '@'
```

Comment:
Can we make it clear that the version will appear always at the end of the string?

Reply:
If the name contains a @ character and dont have version number (I dont know if it is possible...). For example "random@pkg" the name captured would be "random" and the version number would be "pkg".
Maybe a helper string function could be created that uses regex to detect this kind of "{package_name}@{version_number}" form. 
Maybe something like this one:
^(.+)@((?:[0-9]+\.)?(?:[0-9]+\.)?(?:[0-9]+))
The first capture group is the package name and the second capture group is version (supports major, major.minor and major.minor.patch forms)

Reply:
The version is properly extracted in another section. 
I only need to remove everything after `@` if it's a version.

I make the assumption that in the remote case we have a `@` and a version, it will at the end. 
I'll correct this section.